client/electron/index.ts

```diff
@@ -498,6 +501,12 @@ function main() {
     mainWindow?.webContents.send('outline-ipc-push-clipboard');
   });
 
+  // Fetches dynamic key config from a remote URL.
+  ipcMain.handle(
+    'outline-ipc-fetch-config',
```

Comment:
Perhaps rename this to `outline-ipc-fetch-resource` to align with the function name and avoid confusion.

You'll need to change in the caller as well.